rsconnect/main.py

```diff
@@ -1325,6 +1324,98 @@ def deploy_quarto(
         ce.verify_deployment()
 
 
+# noinspection SpellCheckingInspection,DuplicatedCode
+@deploy.command(
+    name="tensorflow",
+    short_help="Deploy TensorFlow models to Posit Connect [v2024.05.0+].",
+    help=(
+        "Deploy a TensorFlow model to Posit Connect. Requires Posit Connect 2024.05.0 or later."
+        "\n\n"
+        "DIRECTORY is the path containing a TensorFlow model."
```

Comment:
Should `DIRECTORY` be in all caps here? If this is just a convention I'm unaware of, ignore this.

Reply:
The all-caps is a convention we used for Quarto content commands. Its presence feels a little redundant, but folks found it helpful during early testing.